Sources/swift-build/main.swift

```diff
@@ -72,6 +72,10 @@ do {
                         
         case .Fetch:
             try fetch(try directories().root)
+        
+        case .Update:
+            try rmtree(try directories().root, "Packages")
+            try fetch(try directories().root)
```

Comment:
It might be better to do the same way as `--clean`.

```swift
case .Update:
    try rmtree(try directories().root, "Packages")
    fallthrough
case .Fetch:
    try fetch(try directories().root)
```